Add new front-end documentation (#225)
appsignal/appsignal-docs: source/front-end/index.html.md

---
title: "AppSignal for JavaScript"
---

> AppSignal has a first-class, hosted solution for catching errors from front-end JavaScript applications and sending them to AppSignal. An `npm` library for catching JavaScript errors is available for all your front-end needs.

Reviewer: ```suggestion
AppSignal has a first-class, solution for catching errors from front-end JavaScript applications and sending them to AppSignal. An `npm` library for catching JavaScript errors is available for all your front-end needs.
```

Hosted could imply that we host the JS files (I believe hosted here means the endpoint), to prevent confusion, maybe remove the word hosted here.

> This is a __Beta__ implementation, which means:

Reviewer 2: I called this early access in https://github.com/appsignal/appsignal-server/pull/4630, need to decide on one of the two.

Author: We decided on Beta, so I've left this as-is for now.

Reviewer 3: URL is error-handling.html
also in navigation it is error-handling 
but title is error catching: inconsistent, should be change one?

---
title: "AppSignal for JavaScript"
---

> AppSignal has a first-class, solution for catching errors from front-end JavaScript applications and sending them to AppSignal. An `npm` library for catching JavaScript errors is available for all your front-end needs.

Reviewer 3: ```suggestion
AppSignal now has an amazing solution for catching errors from front-end JavaScript applications and sending them to AppSignal. An `npm` library for catching JavaScript errors is available for that. 
```

Reviewer 3: We never use 'first class' anywhere as a word. 
maybe it is too soon to call it amazing? but first class is not on brand, imho

Reviewer: It was a second-class feature, which we fixed. I think it's too soon to call it amazing.

Author: Let's be bold and use "amazing" :)
